topics/imaging/tutorials/multiplex-tissue-imaging-TMA/tutorial.md

```diff
@@ -0,0 +1,421 @@
+
+
+![phenoWF](../../images/multiplex-tissue-imaging-TMA/ex2_phenotypeWF.png "Example of a phenotype workflow compatible with Scimap. 'Pos' means that the marker must be positive to be classified as the respective phenotype. 'Anypos' means any, but not necessarily all, of the listed markers can be positive to call the respective phenotype.")
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. {% tool [Single Cell Phenotyping](scimap_phenotyping) %} with the following parameters:
+>
+>    - {% icon param-collection %} *"Select the input anndata"*: Output of **Convert MCMICRO output to Anndata**
+>    - {% icon param-file %} *"Select the dataset containing manual gate information"*: (Optional) manually determined gates in CSV format. Gates will be determined automatically using a GMM for each marker if this file is not provided
+>    - {% icon param-file %} *"Select the dataset containing gating workflow"*: CSV phenotype workflow (Figure 5.)
+>    - *"Save the GMM gates plots If True"*: `Yes`
+>
+>
+>    > ### {% icon comment %} Limitations of GMM automated phenotyping
+>    >
+>    > When manual gates are not provided, Scimap fits a GMM to determine a threshold between positive and negative cells. This automated gating works well when markers are highly abundant within the tissue, and the data shows a bimodal distribution (Figure 6A.). GMM gating can lead to spurious thresholds, however, when the data does not appear to be bimodal (Figure 6B.). This tends to happen when the marker is not highly abundant in the tissue, so there isn't a large positive population. Markers that have a highly continuous range of intensity, like certain functional markers, can also be problematic with GMM gating. It is recommended to always look at the GMM plots output by Scimap, and validate any potentially spurious gates manually. 
+>    >
+>    > ![GMM](../../images/multiplex-tissue-imaging-TMA/ex2_example_GMMs.png "Scimap automatic gating GMMs for two markers. (A) An example of a marker with a bimodal distribution and a reasonable looking gate. (B) An example of a marker with a unimodal distribution that is not ideal for fitting with a GMM, and would be a candidate for manual validation and gating.")
```

Comment:
You could consider using the `Montage` tool here to show a single large montage image with all of the plots, especially useful since you call out the issue with bimodality, it'd make it easier to get a quick overview.